src/tools/management/commands/update_summaries.py

```diff
@@ -23,12 +23,12 @@ def add_arguments(self, parser):
         )
 
     @transaction.atomic
-    def update_summaries(self, project_id):
-        update_project_summaries(project_id)
+    def update_summaries(self, project_id, skip_test_project):
+        update_project_summaries(project_id, skip_test_project)
 
     def handle(self, *args, **options):
         is_forced = options["force"]
-        include_test_projects = options["test_projects"]
+        skip_test_project = options["test_projects"] is not True
 
         if settings.ENVIRONMENT != "prod" and is_forced is False:
```

Comment:
Can we add `dev` here? That was the original ticket, i.e. making sure dashboard updates happened on dev as well as prod.